app/jobs/set_governing_apo_job.rb

```diff
@@ -57,4 +62,22 @@ def update_druid_count
     bulk_action.update(druid_count_total: pids.length)
     bulk_action.save
   end
+
+  def open_new_version(object, log)
+    if DorObjectWorkflowStatus.new(object.pid).can_open_version?
+      begin
+        vers_md_upd_info = {
+          :significance => 'minor',
+          :description => 'Set new governing APO',
+          :opening_user_name => current_user.to_s
+        }
+        object.open_new_version({:vers_md_upd_info => vers_md_upd_info})
+      rescue Dor::Exception => e
+        log.puts("#{Time.current} Failed to open new version for #{object.pid} (bulk_action.id=#{bulk_action.id}): #{e}")
+        return
+      end
+    else
+      log.puts("#{Time.current} Unable to open new version for #{object.pid} (bulk_action.id=#{bulk_action.id})")
```

Comment:
along the lines of the other two comments about failure counts and logging, i'd just throw an exception here, with the message being `Unable to open new version for #{object.pid}`, and then the enclosing `rescue` for `set_governing_apo_and_index_safely` can handle logging and failure count.